internal/subscriber/ordering.go

```
@@ -43,6 +43,14 @@ func (s *OrderedImplementation) GetSubscription() *subscription.Model {
 	return s.subscription
 }
 
+// GetConsumerLag returns perceived lag for the gievn Subscriber
+func (s *OrderedImplementation) GetConsumerLag() map[string]uint64 {
+
+	lag, _ := s.consumer.GetConsumerLag(s.ctx)
```

why are we ignoring err here..?